### ISSUE TITLE
Is there a way to implement times greater than 24 hours?

### ISSUE THREAD

**Issue by Qazzie5** [Pizzabelly/EasyRP#179]
I was hoping for a way to display a countdown that lasted for 24:15:30:00 but the timestamps only allow for times less than 24:00:00. I don't need 'State' so I was wondering if there's a way to set that to display a variable that can be the countdown.

**Comment by adiamaq** [comment]
Discord by default won't go above 24h as far as I know

**Comment by Qazzie5** [comment]
Yeah that's why I'm asking. For now I just have the end date and time as the 'State' but I'd prefer a countdown timer (even if it only updated hourly) I just can't do that with the integrated Discord one.

**Comment by adiamaq** [comment]
Would you mind if I ask you which operative system are you using? I'm on win10 and did everything correctly, yet Discord won't show on game list or activity. Maybe you're on linux?

**Comment by Qazzie5** [comment]
I'm on win10 as well.
